reference/composedcubemaptexture.md

```diff
@@ -0,0 +1,126 @@
+## ComposedCubeMapTexture
+
+```
+ComposedCubeMapTexture {
+  vrmlField SFNode right   NULL
+  vrmlField SFNode left    NULL
+  vrmlField SFNode top     NULL
+  vrmlField SFNode bottom  NULL
+  vrmlField SFNode front   NULL
+  vrmlField SFNode back    NULL
+}
+```
+
+### Description
+
+The [ComposedCubeMapTexture](#composedcubemaptexture) node defines a cubic texture used for environment mapping. It is composed of exactly six [ImageTexture](imagetexture.md) nodes. If one of the [ImageTexture](imagetexture.md)s is not set, the corresponding side of the cubic texture will be black.
```

Comment:
Telling that the six textures are matching with the cube faces (without entering in details here) here would improve the understanding of the node.